discovery-provider/src/tasks/prune_plays.py

```diff
@@ -0,0 +1,103 @@
+import logging
+import time
+from datetime import datetime
+
+from sqlalchemy import text
+from src.tasks.celery_app import celery
+
+logger = logging.getLogger(__name__)
+
+PLAYS_ARCHIVE_TABLE_NAME = "plays_archive"
+
+PRUNE_PLAYS_QUERY = """
+    with archived as (
+        insert into
+            plays_archive
+        select
+            id,
+            user_id,
+            source,
+            play_item_id,
+            created_at,
+            updated_at,
+            slot,
+            signature,
+            :current_timestamp
+        from
+            plays
```

Comment:
are we intentionally not doing this for sol plays only? i.e. signature != null

Reply:
yup this is for all plays